code/modules/admin/create_poll.dm

```diff
@@ -73,7 +73,7 @@
 			if(!option)
 				return
 			option = sanitizeSQL(option)
-			var/default_percentage_calc
+			var/default_percentage_calc = 0
```

Comment:
Needed to fix
```
SQL: Incorrect integer value: '' for column 'default_percentage_calc' at row 1
```
but might just be a side-effect of
```
ADMIN LOG: Database schema (4.1) doesn't match the latest schema version (4.4), this may lead to undefined behaviour or errors
```
due to using MSO's prepackaged mariadb zip.

Reply:
That happens when the db is running on strict mode and you try to insert a non-integer to an int column.

Reply:
This is fine.